designs/snow-integration.md

```diff
@@ -60,6 +60,9 @@ type SnowMachineConfigSpec struct {
 	// Devices contains a device ip list assigned by the user to provision machines.
 	Devices []string `json:"devices,omitempty"`
 
+	// ContainersVolume provides the configuration options for the containers data storage volumes.
```

Comment:
```suggestion
	// ContainersVolume provides the configuration options for the containers data storage volume.
```

Reply:
We chatted offline. Will keep the volume type from upstream. Whenever there is API change upstream, it should be a breaking change and we will handle it on our end too.